### ISSUE TITLE
Destination Typesense: increase timeout and fix batch size

### ISSUE THREAD

**Issue by juanmvrkemok** [airbytehq/airbyte#24097]
## Environment
- **Airbyte version**:  0.42.0
- **OS Version / Instance**: Windows 10
- **Deployment**: Docker
- **Source Connector and version**: PostgreSQL 1.0.48
- **Destination Connector and version**: Typesense 0.1.0
- **Step where error happened**: Sync job

## Current Behavior
When syncing a large dataset to Typesense the connector times out due to a low timeout config in the client's `connection_timeout_seconds` param. Currently it is set to 2 seconds in the destination.py file and should be at least 1 hour.

Also, the batch size is set to `"type": "string",` in the `spec.json` file when it should be `"type": "integer"`.

## Expected Behavior
Connector should not time out even when syncing large amounts of data.

## Logs
urllib3.exceptions.ProtocolError: ('Connection aborted.', timeout('The write operation timed out'))

## Steps to Reproduce
1. Set up an airbyte connection with typesense as destination
2. Try to sync large amounts of data (millions of records)

**Comment by kod-er** [comment]
@juanmvrkemok Were you able to resolve this issue? I get the same error even with a small dataset.

**Comment by juanmvrkemok** [comment]
@kod-er hey there! I made a fork of the project merged this PR into it: https://github.com/airbytehq/airbyte/pull/18806/commits
this fixed the issue, sadly it never got merged into the main project and it's such a simple change :(

**Comment by mattbiondis** [comment]
> @kod-er hey there! I made a fork of the project and merged this PR into it: https://github.com/airbytehq/airbyte/pull/18806/commits this fixed the issue, sadly it never got merged into the main project and it's such a simple change :(

Wow, thank you so much. I got this precisely same issue. I will try your solution. Thank you so much for this insights. @juanmvrkemok

**Comment by mattbiondis** [comment]
Hello @juanmvrkemok, I've already changed all files as you did and already did docker compose down and docker compose up, but still got timeout. Do you have any idea?
Do I have to clone the official repo to my repo and then edit that repo?

**Comment by juanmvrkemok** [comment]
Yeah i believe so, for those changes to work i made a custom connector
following airbyte docs, so instead of using the latest version of the
official typesense connector, i have my own version

**Comment by octavia-squidington-iii** [comment]
At Airbyte, we seek to be clear about the project priorities and roadmap. This issue has not had any activity for 365 days, suggesting that it's not as critical as others. It's possible it has already been fixed. It is being marked as stale and will be closed in 20 days if there is no activity. To keep it open, please comment to let us know why it is important to you and if it is still reproducible on recent versions of Airbyte.
